tests/tests.js

```diff
@@ -1402,6 +1402,23 @@ const unicodeSetFixtures = [
+		options: TRANSFORM_U
```

Comment:
```suggestion
		pattern: '\\P{ASCII}',
		expected: '[\\x80-\\u{10FFFF}]',
		options: { unicodeSetsFlag: "transform", unicodePropertyEscapes: "transform" }
```
Updated the test case so that we don't have to update it again after every new Unicode major releases.